man/docgen

```diff
@@ -275,6 +302,21 @@ class Parameter:
 
         return result
 
+    def help_output(self, indent):
+        result = self.name
+
+        if self.args:
+            result += " " + self.args
+
+        result += " " * (indent - len(result))
+
+        result += textwrap.fill(self.text, width = (90 - indent),
```

Comment:
@fabiangreffrath Should we wrap help text for console output? Maybe we should write shorter help lines instead 😉

Reply:
Help text should fit into one console line ideally. Do you have an example of a help text line that doesn't fit?

Reply:
Oops missed your answer.
This is about a response to the `-help` command. I just generate our CMDLINE documentation, but with selected categories and parameters. So we have a lot of long lines here with multiple sentences.: [help.txt](https://github.com/fabiangreffrath/woof/files/9102301/help.txt)